app/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:23.3.0'
-    compile 'com.android.support:design:23.3.0'
+    compile 'com.android.support:appcompat-v7:26.+'
+    compile 'com.android.support:design:26.+'
```

Comment:
I'm generally a fan of using the concrete version number. At minimum, it gets rid of the Android Studio warning :)

Reply:
Hi bro,
so, what do I do now? should I update the .yml config? after it what should I do to get this pull accepted?

Reply:
Oh, just like, make these version numbers concrete. Meaning that instead of saying `26.+`, you would say `26.0.2` or whatever the actual version is. Otherwise, Google could push a new minor version of the lib and possibly break the CI or something.

Reply:
We still need to make this change.